examples_utils/notebook_logging/README.md

```diff
@@ -17,8 +17,9 @@ This will startup background processes that log and upload system, IPU and noteb
 
 To stop all logging/uploading, run:
 ```
-GCLogger.stop_logging()
+`!export STOP_GC_LOGGING=TRUE`
 ```
+from any cell in the notebook
 
```

Comment:
Have you tested this? I think the `!` runs in a subprocess which means that it does not carry over to the parent environment (where we want the logging to be stopped)

Reply:
Yh I changed it up quite a bit, now using a call from within the kernel that actually unloads and stops the logger extension (line magic % thing)